part-spec/sensor.json

```diff
@@ -0,0 +1,240 @@
+                    "examples": [
+                        "capacitive MEMS",
+                        "piezoresistive",
+                        "piezoelectric"
+                    ],
+                    "type": "string"
+                },
+                "accelerationRanges": {
+                    "description": "range of force that accelerometer can measure",
+                    "examples": [
+                        "2",
+                        "4",
+                        "8",
+                        "16"
+                    ],
+                    "comment": "units of g",
+                    "type": "number"
+                },
+                "accelerationSensitivity": {
+                    "description": "smallest change in force the accelerometer is able to capture (typical). Corresponds to levels in acceleration ranges",
+                    "examples": [
+                        "16000",
+                        "8000",
+                        "4000",
+                        "2000"
+                    ],
+                    "comment": "units of LSB/g; conditions can include acceleration ranges and output resolution",
+                    "type": "number",
+                    "$ref": "definitions.json#/conditionalProperty"
+                },
+                "accelerationSensitivityOverTemperature": {
+                    "description": "accelerometer sensitivity change over temperature",
+                    "comment": "units of percentage per degree celsius (%/Celcius)",
+                    "$ref": "definitions.json#/value"
+                },
+                "axis": {
+                    "description": "number of axes of acceleration measurement",
+                    "type": "number"
+                },
+                "zerogOffset": {
+                    "description": "output of the accelerometer when no acceleration is applied",
+                    "comment": "units of g-force (g)",
+                    "$ref": "definitions.json#/value"
+                },
+                "zerogOffsetOverTemperature": {
+                    "description": "accelerometer zero-g offset change over temperature",
+                    "comment": "units of milli g-force per degree celsius (mg/Celcius)",
+                    "$ref": "definitions.json#/value"
+                },
+                "outputType": {
+                    "description": "measurement output type",
+                    "enum": [
+                        "analog",
+                        "digital"
+                    ],
+                    "type": "string"
+                },
+                "outputResolution": {
+                    "description": "output resolution of acceleration measurement",
+                    "comment": "units of bits",
+                    "type": "number"
+                },
+                "interface": {
+                    "description": "interface(s) for communication to host",
```

Comment:
If we want to support multiple interfaces, we should take this as an array of strings rather than as a stirng.

Reply:
changed to array

Reply:
With an array type we need to specify the type of the inner element, and it looks like we do that via the "items" field: https://github.com/edatasheets/digital-datasheets/blob/main/part-spec/power.json#L133

Please add that field here and throughout where relevant.